### ISSUE TITLE
Fix all infos & warnings from istioctl analyze command

### ISSUE THREAD

**Issue by strekm** [kyma-project/kyma#11686]
**Description**

Clear out an output of `istioctl analyze -n kyma-system` and `istioctl analyze -n kyma-integration` commands. At this moment there are bunch of infos about wrongly named ports and few warnings related to annotations. 
[kyma-integration.txt](https://github.com/kyma-project/kyma/files/6815584/kyma-integration.txt)
[kyma-system.txt](https://github.com/kyma-project/kyma/files/6815585/kyma-system.txt)

**Reasons**

Improve visibility of istio related issues, improve troubleshooting & debugging

**Attachments**
[kyma-integration.txt](https://github.com/kyma-project/kyma/files/6815588/kyma-integration.txt)
[kyma-system.txt](https://github.com/kyma-project/kyma/files/6815589/kyma-system.txt)

**Comment by mjakobczyk** [comment]
It seems that some of the port names connected with `Tracing` can not be changed on Kyma side as they are defined in [`jaeger-operator` itself](https://github.com/jaegertracing/jaeger-operator/blob/master/pkg/service/collector.go#L62).
As a result, following infos would not be addressed:
```bash
Info [IST0118] (Service tracing-jaeger-agent.kyma-system) Port name config-rest (port: 5778, targetPort: 5778) doesn't follow the naming convention of Istio port.
Info [IST0118] (Service tracing-jaeger-collector-headless.kyma-system) Port name c-tchan-trft (port: 14267, targetPort: 14267) doesn't follow the naming convention of Istio port.
Info [IST0118] (Service tracing-jaeger-collector.kyma-system) Port name c-tchan-trft (port: 14267, targetPort: 14267) doesn't follow the naming convention of Istio port.
```

**Comment by cnvergence** [comment]
For duplicate certificate in the application-connector gateway, we won't push any changes
```bash
Warning [IST0138] (Gateway kyma-system/kyma-gateway-application-connector) Duplicate certificate in multiple gateways [kyma-system/kyma-gateway-application-connector kyma-system/kyma-gateway] may cause 404s if clients re-use HTTP2 connections.
```
Please see the explanation [here](https://github.com/kyma-project/kyma/pull/11678#discussion_r691354677)

**Comment by cnvergence** [comment]
The same goes for the tracing-operator
```
Warning [IST0107] (Deployment kyma-system/tracing-jaeger) Misplaced annotation: sidecar.istio.io/inject can only be applied to Pod
Warning [IST0107] (Deployment kyma-system/tracing-jaeger) Misplaced annotation: sidecar.istio.io/rewriteAppHTTPProbers can only be applied to Pod
```
Currently it is not possible to properly set those annotations

**Comment by cnvergence** [comment]
I have opened the issue in the jaeger-operator helm chart
https://github.com/jaegertracing/helm-charts/issues/355

**Comment by cnvergence** [comment]
Port issue can also be tracked here 
- https://github.com/jaegertracing/helm-charts/issues/344

**Comment by cnvergence** [comment]
PRs to jaeger-operator were merged and released with 1.34.0, but we are blocked by this issue 
- https://github.com/jaegertracing/helm-charts/issues/351

**Comment by cnvergence** [comment]
Helm charts of jaeger-operator were fixed
- https://github.com/jaegertracing/helm-charts/issues/351#issuecomment-1153547037

This issue can be closed after bumping jaeger-operator to 1.34.0 version

**Comment by johgoe** [comment]
```
Warning [IST0107] (Deployment kyma-system/tracing-jaeger) Misplaced annotation: sidecar.istio.io/inject can only be applied to Pod
Warning [IST0107] (Deployment kyma-system/tracing-jaeger) Misplaced annotation: sidecar.istio.io/rewriteAppHTTPProbers can only be applied to Pod
Info [IST0118] (Service kyma-system/monitoring-kube-state-metrics) Port name metrics (port: 8081, targetPort: 8081) doesn't follow the naming convention of Istio port.
Info [IST0118] (Service kyma-system/telemetry-operator-webhook) Port name  (port: 443, targetPort: 9443) doesn't follow the naming convention of Istio port.
Info [IST0118] (Service kyma-system/tracing-jaeger-collector-headless) Port name otlp-grpc (port: 4317, targetPort: 4317) doesn't follow the naming convention of Istio port.
Info [IST0118] (Service kyma-system/tracing-jaeger-collector-headless) Port name otlp-http (port: 4318, targetPort: 4318) doesn't follow the naming convention of Istio port.
Info [IST0118] (Service kyma-system/tracing-jaeger-collector) Port name otlp-grpc (port: 4317, targetPort: 4317) doesn't follow the naming convention of Istio port.
Info [IST0118] (Service kyma-system/tracing-jaeger-collector) Port name otlp-http (port: 4318, targetPort: 4318) doesn't follow the naming convention of Istio port.
Info [IST0118] (Service kyma-system/tracing-metrics) Port name cr-metrics (port: 8686, targetPort: 8686) doesn't follow the naming convention of Istio port.
Info [IST0135] (Deployment kyma-system/tracing-jaeger) Annotation "sidecar.istio.io/inject" has been deprecated in favor of the "sidecar.istio.io/inject" label and may not work in future Istio versions.
Info [IST0135] (Pod kyma-system/alertmanager-monitoring-alertmanager-0) Annotation "sidecar.istio.io/inject" has been deprecated in favor of the "sidecar.istio.io/inject" label and may not work in future Istio versions.
Info [IST0135] (Pod kyma-system/cluster-essentials-pod-preset-webhook-858f8d446f-58mvt) Annotation "sidecar.istio.io/inject" has been deprecated in favor of the "sidecar.istio.io/inject" label and may not work in future Istio versions.
Info [IST0135] (Pod kyma-system/cluster-essentials-pod-preset-webhook-858f8d446f-zwpbf) Annotation "sidecar.istio.io/inject" has been deprecated in favor of the "sidecar.istio.io/inject" label and may not work in future Istio versions.
Info [IST0135] (Pod kyma-system/eventing-nats-0) Annotation "sidecar.istio.io/inject" has been deprecated in favor of the "sidecar.istio.io/inject" label and may not work in future Istio versions.
Info [IST0135] (Pod kyma-system/eventing-nats-1) Annotation "sidecar.istio.io/inject" has been deprecated in favor of the "sidecar.istio.io/inject" label and may not work in future Istio versions.
Info [IST0135] (Pod kyma-system/eventing-nats-2) Annotation "sidecar.istio.io/inject" has been deprecated in favor of the "sidecar.istio.io/inject" label and may not work in future Istio versions.
Info [IST0135] (Pod kyma-system/kiali-78d5479f95-jgvhm) Annotation "sidecar.istio.io/inject" has been deprecated in favor of the "sidecar.istio.io/inject" label and may not work in future Istio versions.
Info [IST0135] (Pod kyma-system/logging-loki-0) Annotation "sidecar.istio.io/inject" has been deprecated in favor of the "sidecar.istio.io/inject" label and may not work in future Istio versions.
Info [IST0135] (Pod kyma-system/monitoring-kube-state-metrics-858cd64f74-xp7bb) Annotation "sidecar.istio.io/inject" has been deprecated in favor of the "sidecar.istio.io/inject" label and may not work in future Istio versions.
Info [IST0135] (Pod kyma-system/monitoring-operator-5566cf78-wxvn2) Annotation "sidecar.istio.io/inject" has been deprecated in favor of the "sidecar.istio.io/inject" label and may not work in future Istio versions.
Info [IST0135] (Pod kyma-system/monitoring-prometheus-istio-server-54cfc5f7b5-5pgst) Annotation "sidecar.istio.io/inject" has been deprecated in favor of the "sidecar.istio.io/inject" label and may not work in future Istio versions.
Info [IST0135] (Pod kyma-system/prometheus-monitoring-prometheus-0) Annotation "sidecar.istio.io/inject" has been deprecated in favor of the "sidecar.istio.io/inject" label and may not work in future Istio versions.
Info [IST0135] (Pod kyma-system/serverless-ctrl-mngr-657b8cc46-mcrtd) Annotation "sidecar.istio.io/inject" has been deprecated in favor of the "sidecar.istio.io/inject" label and may not work in future Istio versions.
Info [IST0135] (Pod kyma-system/serverless-webhook-svc-5c58cd8bcd-vqw2k) Annotation "sidecar.istio.io/inject" has been deprecated in favor of the "sidecar.istio.io/inject" label and may not work in future Istio versions.
Info [IST0135] (Pod kyma-system/telemetry-operator-66f4f47ff7-426qq) Annotation "sidecar.istio.io/inject" has been deprecated in favor of the "sidecar.istio.io/inject" label and may not work in future Istio versions.
Info [IST0135] (Pod kyma-system/tracing-jaeger-679ddc6979-n5969) Annotation "sidecar.istio.io/inject" has been deprecated in favor of the "sidecar.istio.io/inject" label and may not work in future Istio versions.
```
